### DISCUSSION TITLE
Overlayed Swatches not displaying on Product images in catalog view

### DISCUSSION THREAD

**Discussion by 2EyeTec** [breezefront/community#230] [1. Help]
When I hover over a picture of a (configurable) product in the catalog view, the area for the swatches fades in, but it's missing the actual color or text swatches. I don't see any errors in the log or developer tools...
This is the url https://www.2eyetec.com/en/camera-slates.html (not all products are configurable)
Thanks in advance for any help!

**Comment by vovayatsyuk** [comment]
It doesn't work because `jsonSwatchConfig` is empty for all products:

<img src="https://github.com/breezefront/community/assets/306080/e1ed8072-6dac-4a96-ab5c-a91b5ae6ec76" width="645" alt="debugger"/>
